addons/mail/static/src/components/messaging_menu/messaging_menu.scss

```diff
@@ -50,6 +50,7 @@
 
     &:not(.o-mobile) {
         display: flex;
+        flex: 1 0 auto;
```

Comment:
Also please double check all flex we use, because this seems like a recurring issue. If checking for `display: flex` is too complex, at least check for those with `flex-flow: column;` which seems to be a more limited subset for this issue to manifest.

Reply:
As said in previous comment, I can't just check each case in the code because this is a strange Safari behaviour that can't be deduced by intended use of flex rules. Only with actual functional use I can see problems. Right now just the messaging menu header is affected.